### DISCUSSION TITLE
Porting existing WPF app

### DISCUSSION THREAD

**Discussion by birojnayak** [AvaloniaUI/Avalonia#5367] [Q&A] [answered]
Hi Guys,
This is Biroj, this is a great initiative...
 We have been contributing to various open source projects. Wanted to know if this project helps moving existing WPF applications from Framework to Core and being platform agnostic ? Or this is more for developing new WPF apps ?

**Comment by jp2masa** [comment, marked as answer]
Porting a WPF app to Avalonia is definitely much easier than to other frameworks, because the concepts and APIs are very similar. However, you will most probably find some differences and missing features, but if you have a specific problem or question the community is usually ready to help, so feel free to ask for support here on Discussions, on [gitter](https://gitter.im/AvaloniaUI/Avalonia) or on a Discord server ([Avalonia](https://discord.gg/gzsBrpNzh7) or [.NET (in the avalonia channel)](https://aka.ms/dotnet-discord)).
